src/inspector_agent.cc

```diff
@@ -575,15 +599,16 @@ void AgentImpl::InstallInspectorOnProcess() {
   env->SetMethod(inspector, "wrapConsoleCall", InspectorWrapConsoleCall);
 }
 
-String16 ToProtocolString(v8::Local<v8::Value> value) {
+std::unique_ptr<StringBuffer> ToProtocolString(v8::Local<v8::Value> value) {
   if (value.IsEmpty() || value->IsNull() || value->IsUndefined() ||
       !value->IsString()) {
-    return String16();
+    return StringBuffer::create(StringView());
   }
   v8::Local<v8::String> string_value = v8::Local<v8::String>::Cast(value);
-  std::basic_string<uint16_t> buffer(string_value->Length(), '\0');
-  string_value->Write(&buffer[0], 0, string_value->Length());
-  return String16(buffer);
+  size_t len = string_value->Length();
+  std::basic_string<uint16_t> buffer(len, '\0');
+  string_value->Write(&buffer[0], 0, len);
+  return StringBuffer::create(StringView(buffer.c_str(), len));
```

Comment:
For my own curiosity, is `.c_str()` rather than `.data()` necessary here if you also pass `len`?